autospec/buildreq.py

```diff
@@ -261,6 +261,44 @@ def parse_cargo_toml(filename):
             add_requires(cdep)
 
 
+def _get_desc_field(field, desc):
```

Comment:
Oh look at us starting to try and indicate private functions, I should start adopting this pattern X(.